mlflow/mleap.py

```diff
@@ -267,7 +266,15 @@ def _handle_py4j_error(reraised_error_type, reraised_error_text):
     """
     traceback.print_exc()
     tb = sys.exc_info()[2]
-    reraise(reraised_error_type, reraised_error_type(reraised_error_text), tb)
+    exception = reraised_error_type(reraised_error_text)
+
+    try:
+        if exception.__traceback__ is not tb:
+            raise exception.with_traceback(tb)
+        raise exception
+    finally:
+        exception = None
+        tb = None
```

Comment:
@rahulporuri What do you think about defining `reraise` somewhere and using it in `mleap.py` and `scoring_server/__init__.py`

Reply:
that sounds like a good idea but i won't be able to make that change before the weekend. I hope that's okay.

Reply:
That's totally fine :)